Add  further content for SIGABRT signal handling

SueChaplain opened the pull request:
- update "Switching to OpenJ9" topic
- udpate signal handling topic
- tweak what's new to explain the work is for compatibility

Closes: #590

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

SueChaplain commented:
@babsingh - please can you check the additional changes I made.

babsingh commented on docs/cmdline_migration.md:
```diff
@@ -90,12 +90,13 @@ These HotSpot command-line options have equivalents in OpenJ9 that are not speci
 |------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
 | ![Start of content that applies only to Java 8 (LTS)](cr/java8.png) [`-Djava.lang.string.substring.nocopy=true`](djavalangstringsubstringnocopy.md) |  Avoid String sharing by String.substring(). ![End of content that applies only to Java 8](cr/java_close_lts.png) |
 | [`-Xnuma:none`](xnumanone.md)                                                     | Disable non-uniform memory architecture (NUMA) awareness.       |
+| [`-XXhandleSIGABRT`](xxhandlesigabrt.md) | Force handling of SIGABRT signals to be compatible with HotSpot |
```
```suggestion
| [`-XX:-HandleSIGABRT`](xxhandlesigabrt.md) | Disable handling of the SIGABRT signal to be compatible with HotSpot |
```

babsingh replied:
The PR still has old changes ... `-XX[+|-]handleSIGABRT` ... missing `:` and `H`. Guess ... after accepting my suggestions in the PR, you may have pushed from an outdated local branch. This would have undone the typo fixes.

babsingh commented on docs/openj9_signals.md:
```diff
@@ -69,6 +69,7 @@ Note that certain signals on VM threads cause OpenJ9 to shutdown. An application
 <i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Notes:**
 
 - The use of `SIGRTMIN` is configurable with the `-Xdump:suspendwith=<num>` option.
+- The handling of `SIGABRT` is configurable with the `-XX[+|-]handleSIGABRT` option.
```
```suggestion
- The handling of `SIGABRT` is configurable with the [`-XX:[+|-]HandleSIGABRT`](xxhandlesigabrt.md) option.
```

babsingh commented on docs/openj9_signals.md:
```diff
@@ -88,6 +89,11 @@ Note that certain signals on VM threads cause OpenJ9 to shutdown. An application
 
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:**
+
+- The handling of `SIGABRT` is configurable with the `-XX[+|-]handleSIGABRT` option.
```
```suggestion
- The handling of `SIGABRT` is configurable with the [`-XX:[+|-]HandleSIGABRT`](xxhandlesigabrt.md) option.
```

babsingh commented on docs/openj9_signals.md:
```diff
@@ -124,6 +130,10 @@ All mechanisms can be disabled by using the `-Xrs` option. However, only structu
 
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Notes:**
+
+- The handling of `SIGABRT` is configurable with the `-XX[+|-]handleSIGABRT` option.
```
```suggestion
- The handling of `SIGABRT` is configurable with the [`-XX:[+|-]HandleSIGABRT`](xxhandlesigabrt.md) option.
```

babsingh commented on docs/openj9_signals.md:
```diff
@@ -148,6 +158,7 @@ All mechanisms can be disabled by using the `-Xrs` option. However, only structu
 
 - VM performance is affected if you install a signal handler for SIGTRAP (5) or SIGRECONFIG (58) because these signals are used for internal control purposes.
 - If you want to generate floating point exceptions, use the following call in your code to generate a `SIGFPE` signal: `fp_trap( P_TRAP_SYNC)`. Although you can use the C compiler `-qflttrap` setting to generate `SIGTRAP` signals to trap floating point exceptions, this mechanism can affect the JIT compiler.
+- The handling of `SIGABRT` is configurable with the `-XX[+|-]handleSIGABRT` option.
```
```suggestion
- The handling of `SIGABRT` is configurable with the [`-XX:[+|-]HandleSIGABRT`](xxhandlesigabrt.md) option.
```

SueChaplain commented:
Thanks @babsingh  - amazing how easy it is to cut and paste errors everywhere ;-) I updated the typo in table of contents where I had picked the bad "handleSIGABRT" option. I also fixed the one above, which also had a lower case "h".

pmhayward commented:
Changes merged 'as-is' and typos fixed separately.
